_queue/template.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
```

Comment:
I would like to provide a place for dev to put robot testcase definition to allow me to triage them and implement them. Maybe it needs a better name?